src/http.ts

```diff
@@ -103,22 +103,25 @@ export enum HTTPRoutes {
   chromeFunction = '/chrome/function?(/)',
   chromePdf = '/chrome/pdf?(/)',
   chromePerformance = '/chrome/performance?(/)',
-  chromeScrape = '/chrome/scrape?(/)',
+  chromeScrape = '/scrape?(/)',
   chromeScreenshot = '/chrome/screenshot?(/)',
+  chromeSiteDownload = '/chrome/site-download?(/)',
```

Comment:
Since we return a variadic payload, maybe we can call it `/chrome/archive` or something? Thinking about getting this route more concise.